### ISSUE TITLE
Add skin tones to the multi-person emojis

### ISSUE THREAD

**Issue by Randl** [Ranks/emojione#322]
As for now, the only option for multi-person emojis like family or couple with heart is generic skin tone. I'd be happy to see skin tones for these emojis just like for all the others.

**Comment by caseyahenson** [comment]
@Randl Things are definitely moving in this direction so I wouldn't be surprised to see this happen within the next couple of Unicode releases.

**Comment by Randl** [comment]
@caseyahenson 
2.2.1 [Multi-Person Groupings](http://www.unicode.org/reports/tr51/#Multi_Person_Groupings)

The basic solution for each of these cases is to represent the multi-person grouping as a sequence of characters—a separate character for each person intended to be part of the grouping, along with characters for any other symbols that are part of the grouping. Each person in the grouping could optionally be followed by an emoji modifier. For example, conveying the notion of COUPLE WITH HEART for a couple involving two women can use a sequence with WOMAN followed by an emoji-style HEAVY BLACK HEART followed by another WOMAN character; **each** of the WOMAN characters **could have an emoji modifier** if desired.

**Comment by JoshyPHP** [comment]
[Unicode's Emoji ZWJ Sequence Catalog for UTR #51 Version 3.0](http://www.unicode.org/Public/emoji/3.0/emoji-zwj-sequences.txt) defines 22 sequences. 1 of them contains no persons, 6 of them contain 2 persons, 6 others contain 3 persons and 9 of them depict 4 persons.

There are 5 skin tones modifiers, which gives us: `6 × 5² + 6 × 5³ + 9 × 5⁴ = 7425` combinations. If Unicode adds support for single-parent families, it will add another cool thousand combinations. That's if we ignore combinations where at least one person is missing a skin modifier, in which case that number almost doubles.

In short, adding support for skin tones in ZWJ sequences will require about 10,000 images.

**Comment by Randl** [comment]
@JoshyPHP Oh, I haven't thought about it that way. But I think, it could be generated with some script, couldn't it?

**Comment by JoshyPHP** [comment]
Last month the Unicode consortium decided to remove support for skin tone modifiers from emoji that depict multiple persons. It is safe to assume there will be no support for multi-skin tones emoji in the foreseeable future.

http://www.unicode.org/L2/L2016/16332-remove-multi-emb.pdf

**Comment by Crissov** [comment]
@JoshyPHP This only affects single characters that feature multiple people, not emoji ZWJ sequences which basically are ligatures, e.g. all (kissing) couple and family emojis except for 💑 U+1F491 / 💏 U+1F48F and 👪 U+1F46A. The emoji font Windows 10 AU supports lots of combinations, but there is no input GUI for all of them.
